examples/film.py

```diff
@@ -35,7 +35,7 @@
 
 from rdflib import BNode, ConjunctiveGraph, URIRef, Literal, Namespace, RDF
 from rdflib.namespace import FOAF, DC
-from six.moves import input
+from builtins import input
```

Comment:
You don't need to import `input` from `builtins`. Anything in `builtins` is automatically imported into every file by default, you can just use `input()` without importing anything.

Reply:
Thanks @ashleysommer, I've removed that line.